Plugins/Onnx/src/MLTrackClassifier.cpp

```diff
@@ -0,0 +1,44 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <cassert>
+#include <stdexcept>
+
+#include "Acts/Plugins/Onnx/MLTrackClassifier.hpp"
+
+// prediction function
+Acts::MLTrackClassifier::TrackLabels Acts::MLTrackClassifier::predictTrackLabel(
+    std::vector<float>& inputFeatures, const double& decisionThreshProb) const {
+  // check that the decision threshold is a probability
+  if ((decisionThreshProb < 0.) || (decisionThreshProb > 1.)) {
```

Comment:
```suggestion
  if ((0 <= decisionThreshProb) and (decisionThreshProb <= 1)) {
```
Readability: this still matches the mathematical notation `0 < x < 1`.